### ISSUE TITLE
Missing timer.h

### ISSUE THREAD

**Issue by douggilliland** [robinsonb5/TG68_MiniSOC#1]
Great project!

I'm trying to build the software and am missing timer.h.

![image](https://user-images.githubusercontent.com/1524110/92338096-f9c06280-f07b-11ea-8135-907d7c090422.png)

Hope you can help!

**Comment by robinsonb5** [comment]
Hi,
Thanks for the report.  Looking at the repo, I see that for some reason the m4k branch is set as the default.  That's an early 
branch and I can't even remember now what it was for - try the master or dualedge branches - they both have the missing file present and correct.

**Comment by douggilliland** [comment]
Awesome, thanks! Compiles now. Ported to Cyclone IV and Cyclone V cards. [Git hub is here](https://github.com/douggilliland/Retro-Computers/tree/master/68000/TG68_AMR).

**Comment by robinsonb5** [comment]
Nice - thanks for sharing!

(If you're interested in CPUs you might find the ZPUDemos, MIPSDemos and EightThirtyTwoDemos fun to tinker with - they all have a similar structure to this one.  Also check out the forum at anycpu.org if you haven't already.)

**Comment by douggilliland** [comment]
Very cool stuff. Will look deeper later. I got started in this with Grant Searle's Multicomp project.

I also rolled my own 32-bit CPU with assembler. https://github.com/douggilliland/R32V2020/wiki

Got stuck when it came to trying to making a C compiler. LLVM broke me ;)

Thanks again.

**Comment by robinsonb5** [comment]
Oh very nice!  I decided very early on that life is too short to tangle with GCC or LLVM.
If you don't mind the weird license, I can heartily recommend vbcc as a compiler.  I found its backend interface really nice to work with - being able to use the "generic risc" backend as a starting point, make incremental changes and see the results in compiled code is immensely valuable.

Anyhow, glad the problem's solved - have fun!
